llvmlite/ir/values.py

```diff
@@ -863,14 +863,16 @@ def __init__(self, args=()):
         for name in args:
             self.add(name)
 
+    def _expand(self, name, typ):
+        return name
+
     def add(self, name):
         if name not in self._known:
             raise ValueError('unknown attr {!r} for {}'.format(name, self))
         return super(AttributeSet, self).add(name)
 
-    def __iter__(self):
-        # In sorted order
-        return iter(sorted(super(AttributeSet, self).__iter__()))
+    def _to_list(self, typ):
+        return list(map(lambda a: self._expand(a, typ), iter(sorted(self))))
```

Comment:
Could this be converted to?

```suggestion
        return [self.expand(i, typ) for i in sorted(self)]
```